ci/steps.yml

```
@@ -175,13 +175,33 @@ steps:
     displayName: Build accelbubble example application to test for android
 
   ##----------------------------------------------------
-  # determine Windows build system
+  # Cache Powershell Modules in $MODULES_FOLDER
+  - task: Cache@2
+    inputs:
+      key: '"pwsh modules" | "$(WINDOWS_VM_IMAGE)"'
```

Maybe it would make more sense here to use the date instead of the name of the image? Something like:
```suggestion
      key: '"pwsh modules" | "$[format('{0:MMyyyy}', pipeline.startTime)]"'
```
This would invalidate the cache once at the beginning of every month, so our copy of Pscx could be at most one month behind the latest version.

Citation for date formatting: https://docs.microsoft.com/en-us/azure/devops/pipelines/process/expressions?view=azure-devops#format

I'm not too sure about the quoting/escaping of quote characters here.